services/src/main/java/org/exoplatform/addons/gamification/GamificationConstant.java

```diff
@@ -35,8 +35,10 @@
     String GAMIFICATION_SOCIAL_COMMENT_NETWORK_STREAM                                = "commentOnNetworkStream";
     /**  COMMENT ON SPACE CONTEXT */
     String GAMIFICATION_SOCIAL_COMMENT_SPACE_STREAM                                  = "commentOnSpaceStream";
-    /**  COMMENT BY A USER */
-    String GAMIFICATION_SOCIAL_COMMENT_ADD                                           = "addComment";
+    /**  COMMENT BY A USER ON NETWORK STREAM*/
+    String GAMIFICATION_SOCIAL_COMMENT_ADD_NETWORK_STREAM                            = "addComment";
+    /**  COMMENT BY A USER ON SPACE STREAM*/
+    String GAMIFICATION_SOCIAL_COMMENT_ADD_SPACE_STREAM                              = "addCommentOnSpaceStream";
```

Comment:
We have here 4 events : commentOnNetworkStream, commentOnSpaceStream, addComment, addCommentOnSpaceStream.
Can you change event name value to have something more readable ?
We should have : receiveCommentOnSpaceStream, receiveCommentonNetworkStream, addCommentOnSpaceStream, receiveCommentOnSpaceStream

Reply:
Okay